src/actions/databases.js

```diff
@@ -1,4 +1,4 @@
-const getDB = require('remote').require('./src/db').getDB;
+import { getDBSession } from '../db';
```

Comment:
I didn't like the name either. I had tought in name it `getDBConnection` as well but actually it is more than only a database connection. It is the sqlectron-db that is a wrap over knex.js. In the end I chose Session inspired on [nhinbernate](http://nhibernate.info/doc/nh/en/index.html#quickstart-playingwithcats).

@esnunes what do you think?